apps/www/pages/open-source/index.tsx

```diff
@@ -56,103 +93,121 @@ const OpenSource = () => {
         }}
       />
       <DefaultLayout>
-        <div className="text-scale-1200 container mx-auto">
-          <SectionContainer>
-            <div className="container">
-              <div className="flex items-center mb-16">
-                <div className="col">
-                  <h1 className="mb-10 text-4xl font-medium">Open source</h1>
-                  <p className="max-w-xl">
-                    Supabase is an open source company, supporting existing open source tools and
-                    communities wherever possible.
-                  </p>
-                </div>
-              </div>
-              <h2 className="mb-6 text-2xl font-medium">Sponsors</h2>
-              <Sponsors />
+        <SectionContainer>
+          <div className="container">
+            <div className="text-center space-y-2">
+              <span className="text-xs md:text-sm uppercase text-brand-900">
+                The Power of Collaboration
+              </span>
+              <h1 className="text-3xl md:text-4xl">Open Source Community</h1>
+              <p className="max-w-xs sm:max-w-md mx-auto text-sm text-scale-1100">
+                Supabase is an open source company, actively fostering collaboration and supporting
+                existing open source tools and communities.
+              </p>
             </div>
-          </SectionContainer>
+          </div>
+        </SectionContainer>
 
-          <SectionContainer>
-            <div className="">
-              <h2 className="mb-6 text-2xl font-medium">Community Maintainers</h2>
+        <SectionContainer>
+          <div className="text-center space-y-2">
+            <h2 className="text-2xl">Sponsor Appreciation</h2>
+            <p className="max-w-xs sm:max-w-md mx-auto text-scale-1100 text-sm pb-8">
+              Thanks to our sponsors we're able to advance our open source mission. Their support
+              fuels our development, fosters collaboration, and empowers us to shape a more open and
+              accessible future.
+            </p>
+            <div>
+              {tiers.map(
+                (t: any, i: number) =>
+                  !!t.transactions.length && (
+                    <div key={i} className="mt-8">
+                      <h3 className="font-medium mb-6">{t.heading}</h3>
```

Comment:
```suggestion
                      <h3 className="font-mono text-scale-900 mb-6">{t.heading}</h3>
```
Based on Figma this should be mono and slightly darker.
<img width="460" alt="Schermata 2023-06-01 alle 12 29 02" src="https://github.com/supabase/supabase/assets/25671831/5ba74276-38b1-4e8e-acb7-6d2702f84cbb">